<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="textbox_password">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(149, 218, 60);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioB_normal">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>generate  a normal strength password
(6 characters)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioB_hard">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>generate hard strength password
(including 8 )</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="radioB_extrahard">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>generate extra hard strength password
(12 characters)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_generate">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(209, 255, 165);</string>
      </property>
      <property name="text">
       <string> Generate Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
